driver-core/src/main/com/mongodb/connection/ReplyHeader.java

```diff
@@ -187,4 +189,9 @@ public boolean isCursorNotFound() {
     public boolean isQueryFailure() {
         return (responseFlags & QUERY_FAILURE_RESPONSE_FLAG) == QUERY_FAILURE_RESPONSE_FLAG;
     }
+
+    // for unit testing
+    int getOpMsgFlagBits() {
```

Comment:
Probably can ditch the comment - but in the test call this method explicitly rather than via the groovyism.